Gemfile

```diff
@@ -20,7 +20,9 @@ gem 'scoped_search', '>= 4.1.8', '< 5'
 gem 'ldap_fluff', '>= 0.5.0', '< 1.0'
 gem 'apipie-rails', '>= 0.5.17', '< 0.6.0'
 gem 'apipie-dsl', '>= 2.2.6'
-gem 'rdoc'
+# Pin rdoc to prevent psych bundle (https://github.com/ruby/rdoc/commit/ebe185c8775b2afe844eb3da6fa78adaa79e29a4)
```

Comment:
I'm not sure what "psych bundle" means. Perhaps this would be clearer?
```suggestion
# Pin rdoc to prevent updating bundled psych (https://github.com/ruby/rdoc/commit/ebe185c8775b2afe844eb3da6fa78adaa79e29a4)
```